tests/test_utils.py

```diff
@@ -211,6 +211,7 @@ def test_get_structured_config_data_illegal_value(test_cls: Any) -> None:
     assert d["x"] == IllegalType()
 
 
+@pytest.fixture  # type: ignore
```

Comment:
So, I don't understand why this is here.
I asked for it to be removed 4 times or so.
am I missing anything?

Reply:
Oh, I removed it before messing the rebase sorry.